web/Services/LandlordService.cs

```diff
@@ -85,9 +84,19 @@ await using (var transaction = await _dbContext.Database.BeginTransactionAsync(I
         return _dbContext.Landlords.SingleOrDefaultAsync(l => l.Id == id);
     }
 
-    public List<PropertyDbModel> GetListOfProperties(int landlordId)
+    public async Task ApproveLandlord(int landlordId,  BricksAndHeartsUser user)
     {
-        return _dbContext.Properties
-            .Where(p => p.LandlordId == landlordId).ToList();
+        var landlord = _dbContext.Landlords.Single(l =>l.Id == landlordId);

```

Comment:
Generally preferable to use `SingleOrDefaultAsync` - async preferred, and the 'or default' bit makes sure it handles it if the id doesn't exist
```suggestion
        var landlord = await _dbContext.Landlords.SingleOrDefaultAsync(l => l.Id == landlordId);
```